packages/types-terra/src/interfaces.ts

```diff
@@ -50,3 +61,13 @@ export interface TerraEvent<M extends Msg = Msg> {
 export type TerraContractMessage = TerraMessage<WasmMsg>;
 
 export type DynamicDatasourceCreator = (name: string, args: Record<string, unknown>) => Promise<void>;
+
+export interface ISafeWasmApi {
```

Comment:
Can this be `type ISafeWasmApi = typeof WasmAPI`? If we can avoid defining our own types that would be ideal